x-pack/plugin/esql/src/main/antlr/EsqlBaseLexer.g4

```diff
@@ -223,8 +221,6 @@ LEFT_BRACES : '{';
 RIGHT_BRACES : '}';
 
 NESTED_WHERE : WHERE -> type(WHERE);
-NESTED_SORT : {this.isDevVersion()}? SORT -> type(SORT);
-NESTED_LIMIT : {this.isDevVersion()}? LIMIT -> type(LIMIT);
```

Comment:
These cause parsing error since commands don't need to be nested. WHERE is declared here for the STATS avg() WHERE declaration -WHERE inside a command.

Reply:
Got it. Thank you.